lib/lndclient/LndClient.ts

```diff
@@ -631,8 +631,7 @@ class LndClient extends SwapClient {
     if (!this.isConnected()) {
       throw swapErrors.FINAL_PAYMENT_ERROR(errors.UNAVAILABLE(this.currency, this.status).message);
     }
-
-    this.logger.trace(`sending payment with ${JSON.stringify(request.toObject())}`);
+    this.logger.debug(`sending payment of ${request.getAmt()} with hash ${request.getPaymentHashString} to ${request.getDestString()}`);
```

Comment:
Should `getPaymentHashString` be a function call here?

Reply:
Yes absolutely, thanks for catching this.